src/file.ts

```diff
@@ -3517,6 +3519,8 @@ class File extends ServiceObject<File> {
    * Resumable uploads are automatically enabled and must be shut off explicitly
    * by setting `options.resumable` to `false`.
    *
+   * Multipart uploads with retryable error codes will be retried 10 times with exponential backoff.
```

Comment:
Do resumable uploads not retry currently? If they do, probably good to add docs about both.

Reply:
@frankyn Do you know if there is existing documentation on resumable upload retries? I'm struggling to find it. @tritone, good callout for adding this documentation, but I'm not sure it belongs here. Multipart retries happen in this function (`file.save`), so it is documented on the function. If resumable upload retries happen, they happen in another function.